_posts/en/2019-10-15-case-continuous-improvement.md

```diff
@@ -44,7 +44,7 @@ If that sounded a little math-y or jargon-y, we'll walk through it below.
 
 If we graph f(x) = x we will get a line as follows
 
-![Graph of f(x) = x]({{ site.baseurl }}/assets/images/graphx.PNG)
+![This image is a line graph that is tracking how many units produced over short and long periods of time. The chart with an x axis going from 0 to 6 and a y axis going from 0 to 6.  It outputs the Y axis as units produced and X as time.  There is one red straight line on the chart that starts at point 0 and goes up until point x=6, y=6. The graph with the straight line demonstrates that as x increases, y steadily increases.]({{ site.baseurl }}/assets/images/graphx.PNG)
 
```

Comment:
Some rewording recommendations to take or leave :)  "This is image is of a line graph.  It has an x axis going from 0 to 6 and a y axis going from 0 to 6. There is one red straight line on the chart that starts at point 0,0 and goes up until point x=6, y=6." 

I've removed text related to interpretation of the graph (e.g. 'the straight line demonstrates') and also info that is not in the graph (e.g. what the x and y axis represent - as those are described in the text below the image.

Reply:
I've removed text related to the interpretation of the graph to avoid repetition. Thanks ;)